File: programmers/java/lev0/수열과 구간 쿼리 1.java
import java.util.stream.IntStream;
import java.util.Arrays;
class Solution {
    public int[] solution(int[] arr, int[][] queries) {
        Arrays.stream(queries).forEach(row->IntStream.range(row[0], row[1]+1).forEach(i->arr[i]+=1));
        return arr;
    }
}

File: programmers/js/lev0/수열과 구간 쿼리 1.js
function solution(arr, queries) {
  queries.forEach((row) => {
    for (let i = row[0]; i <= row[1]; i++) {
      arr[i] += 1;
    }
  });
  return arr;

  //  queries.forEach(([s, e]) => {
  //     while (s <= e) arr[s++]++;
  // });
  // return arr
}
